### PR TITLE
chore: use plugin-bom 2.249-x to define plugin versions

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,9 +144,39 @@
...
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>logging-interceptor</artifactId>
+        <version>3.14.9</version>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okio</groupId>
+        <artifactId>okio</artifactId>
+        <version>1.17.5</version>
```
Comment (bitwiseman) [inline pom.xml:179]: Replace this with the okhttp-api plugin.  This dependency will also be pulled in by github-branch-source.

Comment (garethjevans) [reply pom.xml:179]: I've tried that, but i still get lots of enforcer errors.  It looks like I need both the plugin and the dependencies defined

Comment (bitwiseman) [reply pom.xml]: @garethjevans 
That probably means that there are some things that loading the okhttp3 library and you're probably still packaging it. Could you double check your hpi-plugin output?

**Review by bitwiseman — APPROVED** [review]

**Comment by jtnord** [comment]
no longer needed

**Pull request closed without merging**